newa/__init__.py

```diff
@@ -614,7 +614,10 @@ class Erratum(Cloneable, Serializable):  # type: ignore[no-untyped-def]
     people_assigned_to: str = field(repr=False)
     release: str = field()
     url: str = field()
-    archs: list[Arch] = field(factory=list)
+    archs: list[Arch] = field(factory=list,  # type: ignore[var-annotated]
```

Comment:
Hi @happz , here I need your help. According to my testing, when loading archs from a YAML file then jira_job.erratum.archs at
https://github.com/RedHatQE/newa/blob/main/newa/cli.py#L691
I get a list of strings while it is supposed to be a list of Arch. I believe the root cause is the missing converter for archs attribute which I am adding above but please review whether my assumption is correct. Functional-wise it seem to work.

Reply:
I believe you are correct, there are several other instances of this approach.